features/dataset-metadata.feature

```diff
@@ -0,0 +1,35 @@
+@issue-85 @dataset-metadata-citation-doi @issue-515
+Feature: Add the metadata schema on dataset page to allow Hypothesis to parse citation DOI
+  As a researcher
+  I want the Hypothesis annotation tool to recognise a GigaDB dataset page url as an alias of its DOI
+  I want all search engines, including google to recognise all GigaDB pages url
+  So that I can correctly cite it irrespective of which website it appears on or I annotate from
+  So that all search engines can crawl and index GigaDB pages
```

Comment:
This feature looks strange with two `I want` and two `So that` lines. I think [features](https://behat.org/en/latest/user_guide/features_scenarios.html#user-guide-features-scenarios-features) should only contain three indented lines.

I'm not sure what is best solution. In issue #85, @rija suggests having ` features/dataset-metadata-citation-doi.feature` file for the Hypothesis acceptance. Maybe you could have a ` features/dataset-metadata-search-engine.feature` file too as I can see more tests being added to such a feature file in the future.

Reply:
A new feature file `features/dataset-metadata-search-engine.feature` is created for search engine features.